src/ctia/stores/es/mapping.clj

```diff
@@ -28,19 +27,12 @@
 (def token
   "A mapping for fields whose value should be treated like a symbol.
   They will not be analyzed, and they will be lowercased."
-  {:type "text"
-   ;; TODO use token and disable fielddata once token analyzer is supported
-   :fielddata true
-   :analyzer "token_analyzer"
-   :search_analyzer "token_analyzer"})
```

Comment:
Here is the summary of the conversation about the `lenght` filter that was used for tokens.
This contribution removes it. We have 3 solutions:
- keep this new behavior which is probably the best solution. Since we remove the fielddata, it will not produce any memory issue. In that case you could remove the `token_len` filter and `token_analyzer`  definitions in the index settings, which are not used any more.
- If we do want to not index too long tokens, it has to be reverted and tokens longer that 255 will not be indexed at all and thus could not be queried, which would be the case for many urls. We could significantly raise that value for users that will look for long urls for instance.
 - replace that `length` filter with a [`truncate` filter](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/analysis-truncate-tokenfilter.html) to enable to search on long urls: `{"type": "truncate", "length": 2048}` while limiting the size indexed value (but preserving its full value in `_source`). For the case of urls, they are in general distinct on first 2048 characters.